msg/OrientationConstraint.msg

```diff
@@ -8,10 +8,20 @@ geometry_msgs/Quaternion orientation
 # The robot link this constraint refers to
 string link_name
 
-# optional axis-angle error tolerances specified
+# Optional symmetric error tolerances specified
```

Comment:
Hmmm, indeed, maybe the word "tolerance" implies that it will be applied symmetrically around the current values. I will change something like you suggested.

Reply:
Maybe this?
```
# Optional tolerance on the three vector components of the orientation error
```